src/Tizen.NUI/src/internal/EXaml/Operation/GatherBindableProperties.cs

```diff
@@ -44,7 +44,10 @@ public void Do()
                 field = type.GetField(fi => fi.Name == propertyName && fi.IsStatic && !fi.IsPublic);
             }
 
-            globalDataList.GatheredBindableProperties.Add(field.GetValue(null) as BindableProperty);
+            if (null != field && null != field.GetValue(null) as BindableProperty)
+            {
+                globalDataList.GatheredBindableProperties.Add(field.GetValue(null) as BindableProperty);
+            }
```

Comment:
as following myroot's review, shouldn't this be changed like the below?
```suggestion
            if (null != field && field.GetValue(null) is BindableProperty value)
            {
                globalDataList.GatheredBindableProperties.Add(value);
            }
```

Reply:
> as following myroot's review, shouldn't this be changed like the below?

DONE